docker-compose.yml

```diff
@@ -133,3 +133,18 @@ services:
     working_dir: /app
     volumes:
       - ./src/terraform:/app
+
```

Comment:
Let's discuss this IRL. I would say it's not necessary. When we talked about docker for the front app I thought just a node container that allows running yarn commands on the frontend in case I do not have yarn installed on my laptop.
If you don't personnally need this, you can just remove this commit and whoever needs it will add it later :man_shrugging:

Reply:
Personally I don't need it. 
Let's remove it so.